packages/classic/src/component/Component.ts

```diff
@@ -183,7 +183,7 @@ export const addComponent = (world: World, component: Component, eid: number, re
 		// if it's an exclusive relation, remove the old target
 		if (relation[$exclusiveRelation] === true && target !== Wildcard) {
 			const oldTarget = getRelationTargets(world, relation, eid)[0];
-			if (oldTarget && oldTarget !== target) {
+			if (oldTarget >= 0 && oldTarget !== target) {
```

Comment:
`undefined  >= 0` correctly evaluates as false, but wouldn't a check for `oldTarget` to not be undefined/null be more explicit and readable?

```suggestion
                        if (oldTarget != null && oldTarget !== target) {
```

Reply:
This might even be expanded to `oldTarget !== null && oldTarget !== undefined && oldTarget !== target`.

Reply:
Good point! I'll make this change tonight.